src/components/biggive-timeline/biggive-timeline.tsx

```diff
@@ -0,0 +1,163 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-timeline',
+  styleUrl: 'biggive-timeline.scss',
+  shadow: true,
+})
+export class BiggiveTimeline {
+  @Element() host: HTMLBiggiveTimelineElement;
+
+  @Prop() spaceBelow: spacingOption = 0;
+
+  @Prop() textColour: brandColour = 'black';
+
+  @Prop() selectedTextColour: brandColour = 'primary';
+
+  @Prop() navigationHighlightColour: brandColour = 'grey-medium';
+
+  @Prop() selectedNavigationHighlightColour: brandColour = 'primary';
+
+  @Prop() buttonBackgroundColour: brandColour = 'white';
+
+  @Prop() buttonIconColour: brandColour = 'primary';
+
+  @Prop() entryBackgroundColour: brandColour = 'white';
+
+  @Prop() entryHighlightColour: brandColour = 'secondary';
+
+  @Prop() entryDateColour: brandColour = 'black';
+
+  @Prop() entryTitleColour: brandColour = 'primary';
+
+  @Prop() entryTextColour: brandColour = 'black';
+
+  currentTab = 0;
+  tabs: Array<string> = [];
```

Comment:
Could we rename to `tabHeadings` or something to reflect how its used? Or `years` to reflect the content?

Reply:
I have updated to tabHeadings, as it may be that we have months in the tab in the future.